esx_service/utils/auth.py

```diff
@@ -191,12 +194,24 @@ def get_privileges(tenant_uuid, datastore):
         error_info, privileges = get_default_privileges()
         return error_info, privileges
 
-def has_privilege(privileges, type):
-    """ Check the privileges has the specific type of privilege set. """
+def has_privilege(privileges, type=None):
+    """ Check the privileges has the specific type of privilege set.
+        if param "allow_create" is None or False, which means check
```

Comment:
What does this "param" mean? I guess it means the param "type". Please clarify this.